.github/workflows/ci.yml

```diff
@@ -112,7 +140,11 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-
+      - name: Run docs_snippet_checker
+        if: steps.changed-files.outputs.docs == 'true'
+        run: |
+          yarn install
```

Comment:
How much time does this yarn install step add?
We might think about caching the yarn dependencies.

Reply:
The stage runs in 12 min which is basically identical to what it was before.  We can definitely consider caching if this grows to become the longest stage.